.checkmarx/config.yml

```diff
@@ -0,0 +1,11 @@
+version: 1
+
+# Checkmarx configuration file
+#
+# https://docs.checkmarx.com/en/34965-68549-configuring-projects-using-config-as-code-files.html
+checkmarx:
+  scan:
+    configs:
+      sast:
+        # Exclude test directories
+        filter: "!Networking/Tests/**,!GlobalTestHelpers/**"
```

Comment:
🎨 Probably should include the same list as the PM apps as well: `"!**/*Tests.*/**,!**/TestHelpers/**,!**/Fixtures/**,!GlobalTestHelpers/**"`

Reply:
I see an explicit period after `Tests` -- `Tests.` as the path, which isn't the case here. Should I adjust?